apps/star_wars/mix.lock

```diff
@@ -0,0 +1,21 @@
+%{"absinthe": {:hex, :absinthe, "1.1.8", "b5563ca13a930ac71b61d2f73de03031742215d78d62e17b6d3536bad6ff91ac", [:mix], []},
```

Comment:
Why do we have this mix.lock in star_wars? ... it looks the same than the other ones.